public/docs/_examples/cb-set-document-title/ts/app/main.ts

```diff
@@ -0,0 +1,20 @@
+// #docregion
+import { bootstrap } from 'angular2/platform/browser';
+import { AppComponent } from './app.component';
+
+// While Angular supplies a Title service for setting the HTML document title
+// it doesn't include this service as part of the default Browser platform providers.
+// As such, if we want to inject it into the components within our application,
+// we have to explicitly provide the Angular service in our top component.
+// #docregion bootstrap-title
+import { Title } from 'angular2/platform/browser';
+
+bootstrap(AppComponent, [ Title ])
+// #enddocregion bootstrap-title
+  .then(
+    () => window.console.info( 'Angular finished bootstrapping your application!' ),
```

Comment:
@ocombe In general, I would agree. But, in this case, I'm using the Browser bootstrap. If I were to use a different bootstrap, I think that's when you would need to change the way the promise is handled?